collector/lib/SysdigService.cpp

```diff
@@ -192,6 +203,11 @@ void SysdigService::Start() {
 
   inspector_->start_capture();
 
+  // trigger the self check process only once capture has started,
+  // to verify the driver is working correctly. SelfCheckHandlers will
+  // verify the live events.
+  self_checks::start_self_check_process();
```

Comment:
One unfortunate side effect is that now it will take longer to start collector. Is it possible to somehow do self-checks asynchronously?

Reply:
Have moved this to a thread now, and it seems to work pretty well. Minor caveat that the logs are now a little out of order (it'll often look like we get the self check events before the process exits.)